app/bundles/LeadBundle/Form/Type/FieldType.php

```diff
@@ -138,6 +138,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ]
         );
 
+        $builder->add(
+            'properties_textarea_template',
+            'yesno_button_group',
```

Comment:
This could be class

https://github.com/symfony/symfony/blob/3.0/UPGRADE-3.0.md#form

Type names are also deprecated. You will now not referencing class name but their full qualified class-name.